### ISSUE TITLE
Shebang syntax highlighting

### ISSUE THREAD

**Issue by uncenter** [nefrob/vscode-just#26]
The extension I used for Just syntax highlighting prior to this one, https://github.com/sclu1034/vscode-just, had support for syntax highlighting of shebang lines: https://github.com/sclu1034/vscode-just/blob/06bd5616532470280946c45fd29e5784069a23b2/syntaxes/just.tmLanguage.json. I would love to see that with this one as well!

**Comment by nefrob** [comment]
Seems reasonable. I'll look at getting this updated.

**Comment by jpfeuffer** [comment]
Unfortunately that did not work for me.
<img width="608" alt="Screenshot 2024-08-14 at 13 52 38" src="https://github.com/user-attachments/assets/b60f3569-7003-40d5-a142-e3b9d16b931f">

Maybe a wrong version was uploaded to the Marketplace?

**Comment by nefrob** [comment]
Hmm I'll investigate. Not sure what's failing here.

Running the extension in test mode looks correct:
<img width="664" alt="Screenshot 2024-08-14 at 8 24 29 AM" src="https://github.com/user-attachments/assets/34140919-8403-44a1-8f96-7e7654b5897b">

**Comment by nefrob** [comment]
Confirmed locally that the vsix file on the release was bad. I can't republish the file to hosted marketplaces without a version change. I'll put out a patch release with the correct file to resolve this sometime soon.

In the meantime I repackaged and updated the release artifact  for `0.5.2` if you want to install via `code --install-extension path/to/downloaded/vscode-just-syntax-0.5.2.vsix`.  Then I'd imagine if you toggle on auto updates it will switch to remote version when the next release comes out.
